packages/db/src/schema/collections.ts

```diff
@@ -0,0 +1,126 @@
+import {
+	pgTable,
+	text,
+	timestamp,
+	jsonb,
+	primaryKey,
+	foreignKey,
+	integer,
+} from "drizzle-orm/pg-core";
+import { relations } from "drizzle-orm";
+import { profiles } from "./profiles.ts";
+import { posts } from "./posts.ts";
+
+export const collections = pgTable(
+	"collections",
+	{
+		slug: text("slug").notNull(),
+		locale: text("locale").notNull(),
+		title: text("title").notNull(),
+		description: text("description"),
+		publishedAt: timestamp("published_at").$default(() => new Date()),
+		meta: jsonb("meta"),
+	},
+	(table) => [primaryKey({ columns: [table.slug, table.locale] })],
+);
+
+export const collectionAuthors = pgTable(
+	"collection_authors",
+	{
+		collectionSlug: text("collection_slug").notNull(),
+		collectionLocale: text("collection_locale").notNull(),
+		authorSlug: text("author_slug").notNull(),
+	},
+	(table) => [
+		primaryKey({
+			columns: [table.collectionSlug, table.collectionLocale, table.authorSlug],
```

Comment:
I think referencing the primary key for `collections`/`posts` with multiple columns will make things difficult. With this key, we'd need to create author entries for every locale that the post exists in - and post versions would complicate that further.

It might be better to add a table for `collection_data` (containing all columns) that references the `collections` table (containing only the slug). Then, `collections` can be used in other relations by only referencing the slug. (likewise for `posts` and `post_data`)

That way any chapter/author relations can point to `posts` by the slug only, while `post_data` contains the locale/version-dependent columns.